x/rollapp/keeper/params.go

```diff
@@ -25,7 +29,35 @@ func (k Keeper) DisputePeriodInBlocks(ctx sdk.Context) (res uint64) {
 	return
 }
 
-func (k Keeper) RegistrationFee(ctx sdk.Context) (res sdk.Coin) {
-	k.paramstore.Get(ctx, types.KeyRegistrationFee, &res)
+func (k Keeper) AliasFeeTable(ctx sdk.Context) (res map[string]sdk.Coin) {
+	k.paramstore.Get(ctx, types.KeyAliasFeeTable, &res)
```

Comment:
why map string instead of map int?

Reply:
also I'd just make this return a struct `type aliasPriceMap struct` which automatically takes care of getting the max length one, so you dont need to do that logic in  GetPriceForAlias

Reply:
> why map string instead of map int?

for some reason protobuf wants string as key.

Reply:
🤔